### ISSUE TITLE
Wrong hint for replacing a closure with 'boolval' function call

### ISSUE THREAD

**Issue by cebe** [kalessil/phpinspectionsea#1383]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Ultimate), 3.0.1  |
| Language level | PHP 5.6 |

This could probably already be fixed by https://github.com/kalessil/phpinspectionsea/issues/1367, but it might be different. As the fix for #1367 is not released yet, I decided to report it anyway, please close if it is already fixed.

Current behaviour
--
![Bildschirmfoto von »2019-08-14 12-24-59«](https://user-images.githubusercontent.com/189796/63015126-75a7a580-be90-11e9-966e-b77ce26bbed5.png)

applying the suggestion will change it to:

![Bildschirmfoto von »2019-08-14 12-25-13«](https://user-images.githubusercontent.com/189796/63015142-7f310d80-be90-11e9-8706-5a89b5b8fbeb.png)

which is incorrect.

Expected behaviour
--

This closure can not be simplified as far as I see.

**Comment by cebe** [comment]
Here is the code to copy for trying:

```php
$emails = $this->list_nl_series_emails($nl_series_id);
// skip if series has no active email
if (count(array_filter($emails, function($email) { return (bool) $email['active']; })) === 0) {
    continue;
}
```

**Comment by kalessil** [comment]
Checked: duplicates #1367
